testsuite/performance_benchmark.sh

```diff
@@ -12,18 +12,24 @@ run_benchmark() {
     HIDE_OUTPUT=$3\
     ENABLE_PRUNER=$4\
     NUMBER_OF_EXECUTION_THREADS=$5\
+    SKIP_MOVE_E2E=1\
     ./testsuite/single_node_performance.py
 }
 
+VIRTUAL_CORES=$(getconf _NPROCESSORS_ONLN)
+
+DEFAULT_THREADS=$(($VIRTUAL_CORES > 64 ? 32 : $VIRTUAL_CORES / 2))
```

Comment:
Would it be helpful to echo these values in case we need to debug (Personally, I don't know enough about _NPROCESSORS_ONLN etc to be very confident that these will always give the right values)